src/main/src/main/java/org/geoserver/catalog/ResourcePool.java

```diff
@@ -1820,8 +1820,8 @@ public void writeStyle( StyleInfo info, Style style ) throws IOException {
      */
     public void writeStyle( StyleInfo info, Style style, boolean format) throws IOException {
         synchronized ( styleCache ) {
-            File styleFile = dataDir().findOrCreateStyleSldFile(info);
-            BufferedOutputStream out = new BufferedOutputStream( new FileOutputStream( styleFile ) );
+            Resource styleFile = dataDir().style(info);
+            BufferedOutputStream out = new BufferedOutputStream( styleFile.out() );
```

Comment:
Formatting style is but an agreement for the sake of uniformity, it doesn't reflect on someone's skills. As I commented before, why don't we just change all of it in one go? We are programmers after all, why should we be doing anything manually that we can program?